mm2src/common/header.rs

```diff
@@ -3,27 +3,26 @@
 //! Sometimes we need to call downstream, from a dependency and into a dependent crate,
 //! such as when calling `mm2::rpc::process_rpc_request` from `common::MarketMakerIt::rpc`.
 //! Here we can use C-like headers and/or constructible slots for that.
+//!
+//! TODO refactor this.
 
-#[cfg(not(feature = "native"))] use crate::mm_ctx::MmArc;
-#[cfg(not(feature = "native"))] use bytes::Bytes;
-#[cfg(not(feature = "native"))] use futures01::Stream;
-#[cfg(not(feature = "native"))] use gstuff::Constructible;
-#[cfg(not(feature = "native"))] use http::request::Parts;
-#[cfg(not(feature = "native"))] use http::Response;
-#[cfg(not(feature = "native"))] use std::future::Future;
-
-#[cfg(not(feature = "native"))] use std::net::SocketAddr;
-
-#[cfg(not(feature = "native"))] use std::pin::Pin;
+use crate::mm_ctx::MmArc;
+use gstuff::Constructible;
+use http::request::Parts;
+use http::Response;
+use serde_json::Value as Json;
+use std::future::Future;
+use std::net::SocketAddr;
+use std::pin::Pin;
 
 /// Access to `rpc::process_rpc_request` defined downstream.
 /// Initialized in `rpc::init_header_slots`.
-#[cfg(not(feature = "native"))]
+
 pub static RPC_SERVICE: Constructible<
```

Comment:
I suggest refactoring the `RPC_SERVICE` somehow in the next iteration. It's just why I left the TODO above.